test/Docbook/basic/xsltsubdir/image/subdir/SConscript

```diff
@@ -0,0 +1,8 @@
+Import('env')
+env = env.Clone(tools=['docbook'])
```

Comment:
Curious. Any reason not to just initialize docbook in the top environment and skip the clone()?

Reply:
I copied the test case init from our build scripts which initialise a complex environment at the top level and then have subscripts with more specialised environments which shall be kept segregated from each other.

It was in this hierarchical configuration that I encountered the issue and thats what I based the test case on.